common/persistence/sql/sql_execution_store.go

```diff
@@ -1007,8 +1007,8 @@ func (m *sqlExecutionStore) CreateFailoverMarkerTasks(
 }
 
 type timerTaskPageToken struct {
-	TaskID    int64
-	Timestamp time.Time
+	TaskID    int64     `json:"TaskID"`    // CAUTION: JSON format is used in replication, this should not be changed without great care
```

Comment:
I assume the json annotations are hard contracts to not mess with but it's good to be explicit

Reply:
_generally_ yeah, I (and most people I've seen) think twice before touching something that has explicit annotations like this.  which is the main reason I'm using it, and to be explicit that "this is used by JSON [and not yaml/etc]".

we do have a number of things in e.g. `types/matching` that have these tags spammed AFAICT for better logging purposes.  which is totally reasonable imo, logs are useful, and it's part of why I think JSON should be hard-avoided for anything important :|